src/main/java/me/totalfreedom/totalfreedommod/command/Command_gtfo.java

```diff
@@ -54,6 +54,9 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         // Rollback
         plugin.rb.rollback(player.getName());
```

Comment:
Because I still want to have the rollback system on TFM to be put to use. Because in the rare event the CoreProtect data was wiped or didn't log it for some strange reason and the server hasn't been restarted, it will be rolled back by TFM. This is extreamly rare but it can happen. Also, again, I still want to make use of the TFM rollback system even if the plugin is loaded or not.  I have the rollback and undo rollback methods present if in the event they are needed.